sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```diff
@@ -207,5 +236,9 @@ static def eachDevelocityExtension(def settings, def publicType, def action) {
 }
 
 static boolean isNotAtLeast(String versionUnderTest, String referenceVersion) {
-    GradleVersion.version(versionUnderTest) < GradleVersion.version(referenceVersion)
+    !isAtLeast(versionUnderTest, referenceVersion)
+}
+
+static boolean isAtLeast(String versionUnderTest, String referenceVersion) {
+    GradleVersion.version(versionUnderTest) > GradleVersion.version(referenceVersion)
```

Comment:
Shouldn't the operator be `>=`?

Reply:
The `isNotAtLeast` function had only `>`, so I simply inverted the logic and kept everything as is

Reply:
I actually changed it to `>=` also for that function as I think the way it was used implied `>=` instead of `>`